src/streams/StreamSettingsScreen.js

```diff
@@ -99,10 +99,18 @@ export default function StreamSettingsScreen(props: Props): Node {
       )}
       <View style={styles.padding}>
         {isAdmin && (
+          // TODO: Group all the stream's attributes together (name,
+          //   description, policies, etc.), with an associated "Edit"
+          //   button that gives a UI for changing those attributes. For the
+          //   grouping, try react-native-paper's `Card`, with a
+          //   ZulipTextButton in its `Card.Actions`. See
+          //     https://callstack.github.io/react-native-paper/card-actions.html
+          //   Or their `Surface`:
+          //     https://callstack.github.io/react-native-paper/surface.html
```

Comment:
It seemed like it might take a while to get `react-native-paper` set up just right, so this stuff isn't in my plan for getting the feature out the door. But I wanted to write it down.